### DISCUSSION TITLE
Problème de chargement

### DISCUSSION THREAD

**Discussion by mpweb01** [PrestaSafe/prettyblocks#230] [General]
Bonjour,
Je suis développeur fullstack. J'ai installé prettyblocks sur un serveur et tout à fonctionné correctement. Je l'ai installé sur un nouveau serveur avec la même base de données mais prettyblocks reste bloqué sur "Chargement". La seule chose qui change entre les 2 serveurs est le faite que sur le serveur qui fonctionne, une requete ajax est jouée : module/prettyblocks/ajax?ajax=true&action=GetStates&zone=displayHome&ctx_id_lang=1&ctx_id_shop=1&ajax_token=azvCvun0Epnp43ExLGdOdP8FD mais sur l'autre serveur, cette requêtes ne se lance jamais d'où le fait que prettyblocks reste en chargement.

Je n'ai aucune erreur dans la console.

Pourriez-vous m'aider ? J'ai tenter de mettre en 777 tous les droits sur le serveur mais ça ne change rien.
Auriez-vous une piste ?
Je vous remercie par avance.
Mickael

**Comment by PrestaSafe** [comment]
Bonsoir, avez vous des erreurs JS dans la console javascript ?

**Comment by mpweb01** [comment]
Je n'ai pas d'erreur mais après plusieurs heures de recherche j'ai identifié le souci.
Mon serveur est derrière un cloudflare et prettyblocks n'est pas compatible avec le rocket loader de cloudflare. J'ai donc du créer une règle pour ne pas activer le rocket loader sur la page. Je ne sais pas s'il faut faire remonter le problème auprès des développeur.
Merci pour votre temps
